tests/unittests/Foundation/NSProgressTests.mm

```diff
@@ -259,6 +277,7 @@ - (void)addParentTo:(NSProgress*)child {
 }
 
 ARM_DISABLED_TEST(NSProgress, SingleParentEvenAcrossThreads) {
+    #if TARGET_OS_WIN32
```

Comment:
And split it up into real testable components?

---
In reply to: [73764865](https://github.com/Microsoft/WinObjC/pull/836#discussion_r73764865) [](ancestors = 73764865)